src/main/java/uk/gov/companieshouse/search/api/controller/EnhancedSearchController.java

```diff
@@ -30,7 +31,7 @@
     @Autowired
     private ApiToResponseMapper apiToResponseMapper;
 
-    private static final String COMPANY_NAME_QUERY_PARAM = "q";
+    private static final String COMPANY_NAME_QUERY_PARAM = "company_name";
```

Comment:
Yes it is, it will not be `q` as there is no single required query, therefore with the various filters we need clear indication of what each filter is.

Reply:
As we add fields it may break out into `company_name_must_contain` or `company_name_should_not_contain` etc..